mk/spksrc.wheel.mk

```diff
@@ -62,7 +62,9 @@ build_wheel_target: $(PRE_WHEEL_TARGET)
 		$(foreach e,$(shell cat $(WORK_DIR)/python-cc.mk),$(eval $(e))) \
 		if [ ! -z "$(CROSS_COMPILE_WHEELS)" ] ; then \
 			$(MSG) "Force cross-compile" ; \
+			cp $(WORK_DIR)/install/$(INSTALL_PREFIX)/$(PYTHON_LIB_DIR)/$(_PYTHON_SYSCONFIGDATA_NAME).py $(PYTHON_SITE_PACKAGES_NATIVE)/../ ; \
 			$(RUN) CFLAGS="$(CFLAGS) -I$(STAGING_INSTALL_PREFIX)/$(PYTHON_INC_DIR) $(WHEELS_CFLAGS)" LDFLAGS="$(LDFLAGS) $(WHEELS_LDFLAGS)" $(PIP_WHEEL) ; \
+			rm $(PYTHON_SITE_PACKAGES_NATIVE)/../$(_PYTHON_SYSCONFIGDATA_NAME).py ; \
```

Comment:
Is this removal really required? This "specializes" `native/python3/work-native` content and prevent it to be re-used for any other architecture.
As a result, `make arch-x64-6.1 arch-aarch64-6.1` no longer works as expected (and cannot be built in parallel either...
Isn't possible to keep architecture specific files in `spk/python3/work-$ARCH-$TCVERSION` and leave `native` as-is?

Reply:
Your review is correct but I have not succeeded in any other way. I tried for a long time without success.
I think it's a pip problem: it compiles in a safe environment but doesn't known cross-compilation.
I known that this solution is dirty. As you write, the cp specializes the work-native, I added the line with rm just to delete the library after pip ran and restore the previous native.
But, in this way it cannot be compiled in parallel.